recipes/protobuf/3.9.x/conanfile.py

```diff
@@ -32,57 +47,98 @@ def _is_clang_x86(self):
 
     def source(self):
         tools.get(**self.conan_data["sources"][self.version])
-        extracted_folder = self.name + "-" + self.version
-        os.rename(extracted_folder, self._source_subfolder)
+        version = re.findall("(([0-9]+\\.)*[0-9]+)", os.path.basename(os.path.basename(self.conan_data["sources"][self.version]["url"])))[0][0]
```

Comment:
why do you need all this complex parse to get the version?

Reply:
The version of the tar.gz is 3.9.1 but the protoc binary has a 3.9.1.0 suffix.
But I'll change the version back to 3.9.1 and append the .0 to the binary.

Reply:
@uilianries Fixed it. Please review